sei_for_cosmovisor

```diff
@@ -0,0 +1,532 @@
+rm -rf sei-chain
+#clone sei-chain
+git clone --depth 1 https://github.com/sei-protocol/sei-chain.git
+cd "sei-chai" && echo "sei-chai" dizinine girilemiyor && sleep 10 && exit 13
+git fetch --tags -f
+git checkout $SEIDVER
+# Build the new version
+make install
+go build -o build/seid ./cmd/seid
+# Checkout the binary for 1.0.xbeta
+# if not right version exit script.
+cd $HOME || { echo "$HOME dizinine girilemiyor"; sleep 10; exit 13;}
+#create cosmovisor upgrade directory
+mkdir -p "$DAEMON_HOME"/cosmovisor/upgrades/$SEIDVER/bin
+#copy new seid version correct upgrade cosmovisor directory
+cp "$HOME"/go/bin/seid "$DAEMON_HOME"/cosmovisor/upgrades/"$SEIDVER"/bin
+if [ ! -f "$DAEMON_HOME/cosmovisor/upgrades/$SEIDVER/bin/seid" ]; then
+    echo "$DAEMON_HOME/cosmovisor/upgrades/$SEIDVER/bin/seid FILE NOT EXIST"
+    read -r -s -n 1 -p "Press any key to EXIT . . ."
+    exit 13
+else
+  SEIDBUILDVER=$("$DAEMON_HOME"/cosmovisor/upgrades/"$SEIDVER"/bin/seid version)
+  if [ "$SEIDBUILDVER" == "$SEIDVER" ]; then
+  	echo "$DAEMON_HOME"/cosmovisor/upgrades/$SEIDVER/bin/seid version $SEIDVER
+  else
+    	echo -e "\e[1m\e[31m2. Error version not match $SEIDVER \e[0m"
+    	echo -e "\e[1m\e[31m2. please check $DAEMON_HOME/cosmovisor/upgrades/$SEIDVER/bin version file \e[0m"
+    	read -r -s -n 1 -p "Press any key to EXIT . . ."
+  	exit 13
+  fi
+fi
+#################################################
+
+
+# Checkout the binary for 1.0.4beta
+#################################################
+SEIDVER=1.0.4beta
+cd $HOME || { echo "$HOME dizinine girilemiyor"; sleep 10; exit 13;}
+#remove old sei-chain directory
+rm -rf sei-chain
+#clone sei-chain
+git clone --depth 1 https://github.com/sei-protocol/sei-chain.git
+cd "sei-chai" && echo "sei-chai" dizinine girilemiyor && sleep 10 && exit 13
+git fetch --tags -f
+git checkout $SEIDVER
+# Build the new version
+make install
+go build -o build/seid ./cmd/seid
+# Checkout the binary for 1.0.xbeta
+# if not right version exit script.
+cd $HOME || { echo "$HOME dizinine girilemiyor"; sleep 10; exit 13;}
+#create cosmovisor upgrade directory
+mkdir -p "$DAEMON_HOME"/cosmovisor/upgrades/$SEIDVER/bin
+#copy new seid version correct upgrade cosmovisor directory
+cp "$HOME"/go/bin/seid "$DAEMON_HOME"/cosmovisor/upgrades/"$SEIDVER"/bin
+if [ ! -f "$DAEMON_HOME/cosmovisor/upgrades/$SEIDVER/bin/seid" ]; then
+    echo "$DAEMON_HOME/cosmovisor/upgrades/$SEIDVER/bin/seid FILE NOT EXIST"
+    read -r -s -n 1 -p "Press any key to EXIT . . ."
+    exit 13
+else
+  SEIDBUILDVER=$("$DAEMON_HOME"/cosmovisor/upgrades/"$SEIDVER"/bin/seid version)
+  if [ "$SEIDBUILDVER" == "$SEIDVER" ]; then
+  	echo "$DAEMON_HOME"/cosmovisor/upgrades/"$SEIDVER"/bin/seid version "$SEIDVER"
+  else
+    	echo -e "\e[1m\e[31m2. Error version not match $SEIDVER \e[0m"
+    	echo -e "\e[1m\e[31m2. please check $DAEMON_HOME/cosmovisor/upgrades/$SEIDVER/bin version file \e[0m"
+    	read -r -s -n 1 -p "Press any key to EXIT . . ."
+  	exit 13
+  fi
+fi
```

Comment:
all of this logic seems to be repeated for each version. Let's make a bash function and pass in the SEIDVER so that we can reduce duplicate code